buildkite/interpolate updated; ability to use numeric default

pda opened the pull request:
Update buildkite/interpolate to get https://github.com/buildkite/interpolate/pull/7 /cc @ticky 

`${VAR:-127}` will now resolve to `127` if `VAR` isn't set.

Delta commit log on buildkite/interpolate:

    *   07f35b4 Merge pull request #7 from buildkite/bash-compliant-substring-parsing Jessica Stokes 2 hours ago
    |\
    | * 24225d9 Update substring parser to comply with Bash's rules Jessica Stokes 10 days ago
    |/
    * 973457f Add an empty go.mod file Lachlan Donald 1 year, 7 months ago
    * cfbed48 Update README to reflect current code Lachlan Donald 1 year, 7 months ago
    * c1c376f Merge pull request #4 from buildkite/expose-identifiers-from-parse-tree Lachlan Donald 2 years, 3 months ago
    * ad3fc26 Add a interpolate.Identifiers() for getting identifiers from a string Lachlan Donald 2 years, 3 months ago

ticky commented:
Out of curiosity, what was the correct incantation to sync this? Or did you manually copy the SHA?

pda commented:
> Out of curiosity, what was the correct incantation to sync this? Or did you manually copy the SHA?

@ticky I ran `go get -u github.com/buildkite/interpolate` in the project, which fetched the latest version and updated `go.mod` and added the new checksum to `go.sum`. I don't think the `-u` was necessary.
